src/Resources/Wacc/Gifts/gifts.ts

```diff
@@ -0,0 +1,33 @@
+import AbstractResource from '../../AbstractResource';
+
+export default class Gifts extends AbstractResource {
+    initialise(): void {
+        this.resourcePath = '/gifts';
+        this.defaultAccess = 'wacc';
+    }
+
+    public all: Function = this.createMethodFromPartialSpec({
+        method: 'GET',
+        path: '',
```

Comment:
These should ideally have a slash as the path please

```suggestion
        path: '/',
```

Reply:
cool updated now